scheduler-actor-reminders/client/player-actor-client.go

```diff
@@ -23,124 +22,48 @@ func main() {
 		panic(err)
 	}
 	defer client.Close()
+	if err := client.Wait(ctx, 15*time.Second); err != nil {
+		log.Fatalf("error waiting for Dapr initialization: %v", err)
+	}
 
-	// Actor ID for the player 'session'
-	actorID := "player-1"
-	deathSignal := make(chan bool)
+	// Implement the actor client stub
+	myActor := &api.ClientStub{
+		ActorID: "player-1",
+	}
+	client.ImplActorClientStub(myActor)
 
 	// Start monitoring actor player's health
-	go monitorPlayerHealth(ctx, client, actorID, deathSignal)
-
-	incReminderCtx, incReminderCancel := context.WithTimeout(ctx, 5*time.Second)
-	defer incReminderCancel()
-	// Start player actor health increase reminder
-	err = client.RegisterActorReminder(incReminderCtx, &dapr.RegisterActorReminderRequest{
-		ActorType: "playerActorType",
-		ActorID:   actorID,
-		Name:      "healthReminder",
-		DueTime:   "10s",
-		Period:    "20s", // Every 20 seconds, increase health
-		Data:      []byte(`"Health increase reminder"`),
+	go monitorPlayerHealth(ctx, myActor)
+
+	//Start player actor health increase reminder
```

Comment:
```suggestion
	// Start player actor health increase reminder
```
nit, otherwise if lint runs it will complain. I think this was just tweaked accidentally with moving things around

Reply:
Oh, yeah, completely unintentional. Thanks!